tomviz/ViewMenuManager.cxx

```diff
@@ -11,7 +11,14 @@
 
 #include <vtkCommand.h>
 #include <vtkGridAxes3DActor.h>
+#include <vtkImageProperty.h>
```

Comment:
I think that the view dark/white dialog is likely to need to grow in the future, and I don't think we want to introduce all of these classes to the ViewMenuManager. I think creating a SliceViewDialog would be preferable where we can change which slice is passed to it would offer better encapsulation, and let us grow this dialog as necessary.